projects/demo-playwright/tests/kit/input-range/input-range.spec.ts

```diff
@@ -0,0 +1,258 @@
+import {
+    TuiDocumentationApiPagePO,
+    tuiGoto,
+    TuiInputRangePO,
+} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+import {TUI_PLAYWRIGHT_MOBILE_USER_AGENT} from '../../../playwright.options';
+
+test.describe('InputRange', () => {
+    let example: Locator;
+    let range: TuiInputRangePO;
+
+    test.use({viewport: {width: 1280, height: 800}});
+
+    test.describe('Keyboard interactions', () => {
+        test.beforeEach(async ({page}) => {
+            await tuiGoto(page, '/components/input-range/API?min=-100&max=100&quantum=5');
+            example = new TuiDocumentationApiPagePO(page).apiPageExample;
+            range = new TuiInputRangePO(example.locator('tui-input-range'));
+        });
+
+        test('pressing Arrow Down decreases LEFT value when LEFT text input is focused', async ({
+            page,
+        }) => {
+            await range.leftTextInput.focus();
```

Comment:
Use `textfield` word to be similar to other page objects (i.e. `leftTextfield`)

https://github.com/taiga-family/taiga-ui/blob/4ee1b099e95381ca866e1088f0f967872e1c5e56/projects/demo-playwright/utils/page-objects/input-slider.po.ts#L3-L4

Reply:
(the same for `rightTextfield`)